packages/svelte/messages/client-warnings/warnings.md

```diff
@@ -56,16 +56,3 @@
 	value === proxy; // always false
 </script>
 ```
-
-In the rare case that you need to compare them, you can use `$state.is`, which unwraps proxies:
```

Comment:
We should probably replace this paragraph with something like "to avoid this situation, either only compare values initialized through `$state` or use `$state.raw` instead, which will not proxify the underlying object"

Reply:
added a note